profiles/dave-hill.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/99537934?s=400&u=9c54d90592659caa2d4af3c0c6f6a8c9fe6e44a7&v=4",
```

Comment:
This link should probably be the image link.

```suggestion
  "image": "https://user-images.githubusercontent.com/99537934/255003766-7311cd18-62ed-4595-9be6-26d05abd1359.png",
```

![image](https://user-images.githubusercontent.com/99537934/255003766-7311cd18-62ed-4595-9be6-26d05abd1359.png)

Reply:
Updated